src/pages/services/aem-assets-view/api/browse-view/index.md

```diff
@@ -0,0 +1,497 @@
+- **resourceSelection** (`object`): an object representing the resource selection.
+  - resources (`array`): an array of selected resources.
+    - id (`string`): selected resource URN.
+    - path (`string`): selected resource path.
+
+**Returns** (`array`) an array of custom action descriptors or an empty array if no custom actions should be added to the ActionBar.
+
+Each array element is a custom action descriptor that is a JSON with the following properties:
+- `id` (`string`): action id, unique within given extension.
+- `label` (`string`): Custom action title.
+- `icon` (`string`): Name of the [React-Spectrum workflow icon](https://react-spectrum.adobe.com/react-spectrum/workflow-icons.html#available-icons).
+- `onClick` (`function`): An activation handler.
+
+**Example:**
+```js
+actionBar: {
+    getActions: ({ context, resourceSelection }) => {
+        return [{
+            'id': 'customId',
+            'icon': 'Form',
+            'label': 'Custom label',
+            'onClick': async () => {
+                // ...
+            }
+        }];
+    },
+}
+```
+
+`getHiddenBuiltInActions({ context, resourceSelection })`
+
+**Description:**  returns an array of [built-in action id](#built-in-actions) that should be hidden in the specified context for the selected resources.
+
+This method is called by the host application to determine which built-in actions are to be hidden.
+The host calls this method once whenever the resource selection changes.
+ 
+Extension code should ensure that this method returns fast because the host application blocks the rendering of the
+ActionBar until the actions' visibility could be determined.
+In particular it is recommended not to use backend server calls in this method.
+
+**Parameters:**
+- context (`string`): current [browsing context](#browsing-context)
+- **resourceSelection** (`object`): an object representing the resource selection
+  - resources (`array`): an array of selected resources.
+    - id (`string`): selected resource URN.
+    - path (`string`): selected resource path.
+
+**Returns** (`array`) an array of action ids that should be hidden from the ActionBar, or an empty array in case no action needs to be hidden
+
+**Example:**
+ ```js
+getHiddenBuiltInActions: ({ context, resourceSelection }) => {
+    return [];
+},
+```
+
+`overrideBuiltInAction({ actionId, context, resourceSelection })`
+
+**Description:**  Return true to indicate the Host should perform the built-in action, false otherwise. 
+
+This method is called by the Host when the user activates one of the built-in actions, before invoking the actual action
+handler. The method returns true if the Extension had performed custom action processing and the Host should not invoke
+built-in action handler. Otherwise the method call returns false, to indicate that the Extension
+had ignored the invocation and the Host should use built-in action handler.
+
+**Parameters:**
+- actionId (`string`): actionId built-in action id.
```

Comment:
Let's link the term "built-in action id" to definition table similar to what we did in getHiddenBuiltInActions